tools/go-cli/go-whisk-cli/commands/action.go

```diff
@@ -52,17 +52,8 @@ var actionCreateCmd = &cobra.Command{
   PreRunE:       setupClientConfig,
   RunE: func(cmd *cobra.Command, args []string) error {
 
-    if len(args) < 2 {
-      whisk.Debug(whisk.DbgError, "Action create command must have at least two arguments\n")
-      errMsg := fmt.Sprintf("Invalid argument(s). An action name and artifact are required.")
-      whiskErr := whisk.MakeWskError(errors.New(errMsg), whisk.EXITCODE_ERR_GENERAL,
-        whisk.DISPLAY_MSG, whisk.DISPLAY_USAGE)
-      return whiskErr
-    } else if len(args) > 2 {
-      whisk.Debug(whisk.DbgError, "Action create command must not have more than two arguments\n")
-      errMsg := fmt.Sprintf("Invalid argument(s): %s.", strings.Join(args[2:], ", "))
-      whiskErr := whisk.MakeWskError(errors.New(errMsg), whisk.EXITCODE_ERR_GENERAL,
-        whisk.DISPLAY_MSG, whisk.DISPLAY_USAGE)
+    if whiskErr := checkArgs(args, 2, 2, "Action create",
+      "An action name and artifact are required."); whiskErr != nil {
```

Comment:
The word `artifact` is no longer part of the `wsk action create -h` help output.  
```
create a new action
Usage:
  wsk action create ACTION_NAME ACTION [flags]

Flags:
  -a, --annotation KEY VALUE   annotation values in KEY VALUE format (default [])
      --copy                   treat ACTION as the name of an existing action
      --docker                 treat ACTION as docker image path on dockerhub
```